src/components/atoms/ButtonNG/ButtonNG.scss

```diff
@@ -0,0 +1,121 @@
+@import "scss/constants.scss";
+
+.ButtonNG {
+  display: inline-block;
+
+  padding: 0.75em;
+  margin: $spacing--xs;
+  width: 100%;
+  max-width: 275px;
```

Comment:
Could we store `275px` as a variable?

Reply:
What's the rationale for it?

Few questions that may help me understand it better:

- Why if it's used only once in one place?
- Doesn't property name `max-width` describes its intent already?
- If that's the only variable substitution  what makes it different from `width` or `padding`?

Reply:
Well, it looks useful to store the main measurements of the component view at the head of a file (like in other component styles files).
Paddings are used just for displaying the block and it's fine to define them at a place. 
And `100%` is a general measure. Otherwise, it could be settled in `scss/constants`

Reply:
K, but since we're starting down this road, I'm starting to reconsider if `max-width` is all we need, there should probably be `min-width` and few others...

Reply:
I'd put those down as OOS and so that @sparkletown/design may have some starting point to provide more input.

Reply:
Added variable @ 187a97d35ba7dea92fa0730f70886a2dc50a5aec